dandi/validate_types.py

```diff
@@ -1,35 +1,53 @@
 from __future__ import annotations
 
 from dataclasses import dataclass
-from enum import Enum
+from enum import Enum, IntEnum
 from pathlib import Path
+from typing import Any
 
 
 @dataclass
 class ValidationOrigin:
-    name: str
-    version: str
-    bids_version: str | None = None
+    name: str  # Validator name
+    version: str  # Validator version
+    standard: str | None = (
+        None  # Standard being validated against # TODO: Enum for the standards??
+    )
+    standard_version: str | None = None  # Version of the standard
 
 
-class Severity(Enum):
+# TODO: decide on the naming consistency -- either prepend all with Validation or not
+class Severity(IntEnum):
     HINT = 1
-    WARNING = 2
-    ERROR = 3
+    INFO = 2  # new/unused, available in linkml
+    WARNING = 3
+    ERROR = 4
+    CRITICAL = 5  # new/unused, linkml has FATAL
 
 
 class Scope(Enum):
     FILE = "file"
     FOLDER = "folder"
+    # Isaac: make it/add "dandiset-metadata" to signal specific relation to metadata
```

Comment:
Need clarification for this.

Reply:
may be forget about it for now -- seems to combine with `ValidationObject` below but we might need to rethink/define some additional levels

Reply:
as for the folder an example could be an unknown to BIDS folder name under `sub-.../ses-.../` folder.  I am not sure ATM if BIDS validator complains and how